youtube_upload/main.py

```diff
@@ -232,7 +231,7 @@ def main(arguments):
     parser.add_option('', '--tags', dest='tags', type="string",
                       help='Video tags (separated by commas: "tag1, tag2,...")')
     parser.add_option('', '--privacy', dest='privacy', metavar="STRING",
-                      default="public", help='Privacy status (public | unlisted | private)')
+                      default="private", help='Privacy status (public | unlisted | private)')
```

Comment:
Also need to change recordingDetails.

Reply:
done sir

Reply:
@kidbit-ui-dev I don't see it as part of the PR.